### ISSUE TITLE
Mealplan badly formatted

### ISSUE THREAD

**Issue by swongts** [swongts/pe#21]
![image.png](https://raw.githubusercontent.com/swongts/pe/main/files/0a99cad8-dd04-4fdc-bc8e-b1ebae33349e.png)

Confusing as add /mealplan is not included in meal plan database category. I was very confused about create /mealplan and add /mealplan just from reading the user guide.

**Comment by nus-se-script** [comment]
# Team's Response
This was a deliberate decision since the action of "adding mealplans" in layman terms is adding the food items under a meal plan into the entry database. As such, in order to reduce duplicate entries inside the feature list, this action was listed directly under the entry database.


# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]



- [x] I disagree

**Reason for disagreement:** I understand this decision since you are technically adding to the entries, but it is confusing for a reader because there is suddenly an add /mealplan. Perhaps you should have explained the relations between these entry, mealplan, food database before explaining the actual features. This command in general is confusing because add usually means you are creating something, and I had quite a lot of difficulty differentiating add and create. 

![image.png](https://raw.githubusercontent.com/swongts/pe/main/files/6577f361-407e-48a4-beb3-a580a3b18699.png)

In fact in this screenshot, "adds a meal plan consisting of existing food items" makes it seem like you are creating a meal plan. I definitely this greatly hindered my ability of understanding the application, and grouping meal plan commands together would have been a better call.

It is not very clear to me that the features under entry database means the feature that makes changes to the entry database, and it also shouldn't be grouped this way especially if you are introducing something that had not been explained or mentioned at all before (meal plan)


-------------------
